### DISCUSSION TITLE
Site Authentication - Keycloak

### DISCUSSION THREAD

**Discussion by JamesRivers** [squidfunk/mkdocs-material#4307] [Q&A]
Has anyone reviewed / talked about the possibility of using something like Keycloak for site authentication and access? 

<img width="772" alt="image" src="https://user-images.githubusercontent.com/8113173/187916010-50c28b96-00f8-4546-81f4-c6c799b3d9e3.png">

I am hosting static site on gh-pages but want a level of control and access authentication on the front end.

**Comment by squidfunk** [comment]
There have been some questions regarding putting authentication in front of documentation sites, but all of them generalize to any other static site, which means it's really beyond the scope of this project. It's probably better to ask on StackOverflow.
